packages/invoices/src/OutboundShipment/DetailView/SidePanel.tsx

```diff
@@ -101,45 +107,28 @@ const RelatedDocumentsSection: FC<SidePanelProps> = () => {
   );
 };
 
-export const SidePanel: FC<SidePanelProps> = ({ draft }) => {
+export const SidePanel: FC = () => {
   const { success } = useNotification();
   const t = useTranslation('distribution');
+  const { data } = useOutbound();
 
   const copyToClipboard = () => {
-    navigator.clipboard.writeText(JSON.stringify(draft, null, 4) ?? '');
+    navigator.clipboard.writeText(JSON.stringify(data, null, 4) ?? '');
     success('Copied to clipboard successfully')();
   };
 
   return (
     <DetailPanelPortal
       Actions={
-        <>
-          {!process.env['NODE_ENV'] ||
-            (process.env['NODE_ENV'] === 'development' && (
-              <DetailPanelAction
-                icon={<CopyIcon />}
-                title={t('dev.log-draft')}
-                onClick={() => {
-                  console.table(draft);
-                  draft.items.forEach(item => {
-                    console.table(item);
-                    Object.values(item.batches).forEach(batch => {
-                      console.table(batch);
-                    });
-                  });
-                }}
-              />
-            ))}
-          <DetailPanelAction
-            icon={<CopyIcon />}
-            title={t('link.copy-to-clipboard')}
-            onClick={copyToClipboard}
-          />
-        </>
+        <DetailPanelAction
+          icon={<CopyIcon />}
+          title={t('link.copy-to-clipboard')}
+          onClick={copyToClipboard}
+        />
       }
     >
-      <AdditionalInfoSection draft={draft} />
-      <RelatedDocumentsSection draft={draft} />
+      <AdditionalInfoSection />
```

Comment:
a minor thought.. the `dev.log-draft` entries could be removed from the translation files now

Reply:
Ah yes! 

I will remember to remove this - for now I'd like to resolve the conflicts and merge as at a guess I'll just cause more conflicts by touching the translation files!